api/src/controllers/document/postDocument.ts

```diff
@@ -34,7 +35,7 @@ class ReqBodyValidator extends Validator<ReqBody> {
 
         this.ruleFor('isReview').notNull();
 
-        this.ruleFor('userId').mustAsync(beAValidUser);
+        this.ruleFor('userId').mustAsync(beAValidUser); // TODO this should match the user creating it
```

Comment:
The user that created the `resumeReview` object?

Reply:
Yeah good point. Not sure that I even want to add this restriction in though. This makes it impossible for the Admin to go in and mess around with things. Just put it down as a quick idea.

Reply:
Sounds good, we can put it aside for now I don't expect anyone to go out and take a token to override someone else's resume review. Also these are creates which aren't as destructive as deletes so, we should be fine